### PR TITLE
Add others rows if there are more than 16 rows amount all tables

### PR DESCRIPTION
**Pull request by alejandrocabriales**
# Ticket

https://trello.com/c/JHKHOrzX/294-feature-budget-summary-navigation

# What solved
- [X] Should have the table show (Others) rows, when there is more subcategory to show, and this should be a resume of the rest.

# Description
Add others row when there is more than 16 rows, and only show 16 rows as max numbers of row

### CODE REVIEW

File: src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx [hunk 1]
```diff
@@ -29,17 +30,19 @@ const FinancesTable: React.FC<Props> = ({ className, breakdownTable, metrics, pe
   const showQuarterly = !isMobile && period === 'Quarterly';
   const showMonthly = desk1440 && period === 'Monthly';
   const arrayMetrics = new Array<number>(iteration).fill(0);
-  const showFooter = true;
-  // Show color for others depending if odd or even
-  const isPair = orderData[`${tables[tables.length - 1]}`]?.length % 2 === 0;
+
+  // Show color for others depending if odd or pair
+  const isPair = showFooterAndCorrectNumber[`${tables[tables.length - 1]}`]?.length % 2 === 0;
```
Comment (YasielCabrera) [inline src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx:35]: please, note here that it should be "isEven", in English we say "odd or even", not "odd or pair"

File: src/stories/containers/Finances/utils/utils.ts [hunk 2]
```diff
@@ -716,3 +717,51 @@ export const sortDataByElementCount = (data: MockData) => {
 
   return sortedData;
 };
+
+export const showOthersFooterRow = (data: MockData) => {
+  let totalRows = 0;
+  Object.keys(data).forEach((key) => {
+    totalRows += data[key].length;
+  });
+  console.log({ totalRows });
```
Comment (YasielCabrera) [inline src/stories/containers/Finances/utils/utils.ts:726]: did you forgot to delete this ` console.log` ?

File: src/stories/containers/Finances/utils/utils.ts [hunk 3]
```diff
@@ -716,3 +717,51 @@ export const sortDataByElementCount = (data: MockData) => {
...
+  const firstElements: Record<string, QuarterlyBudget[]> = {};
+  const totalRows = Object.keys(data).reduce((acc, key) => acc + data[key].length, 0);
+  // Si la cantidad total de filas es menor o igual a 16, no necesitas hacer cambios
+  if (totalRows < maxRows) {
+    return data;
+  }
+
+  // Save the first element of each table
+  Object.keys(data).forEach((key) => {
+    firstElements[key] = [data[key][0]];
```
Comment (YasielCabrera) [inline src/stories/containers/Finances/utils/utils.ts:754]: are we 100% sure that `data[key]` will always be an array with at least 1 element? If not, we should prepare to receive empty arrays here, if that happens we could have an error here

**Review by YasielCabrera — CHANGES_REQUESTED** [review]

File: src/stories/containers/Finances/utils/utils.ts (hunk 3 above)
Comment (alejandrocabriales) [reply src/stories/containers/Finances/utils/utils.ts:754]: You are right

File: src/stories/containers/Finances/utils/utils.ts (hunk 2 above)
Comment (alejandrocabriales) [reply src/stories/containers/Finances/utils/utils.ts:726]: yes, done

File: src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx (hunk 1 above)
Comment (alejandrocabriales) [reply src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx:35]: you are right, done

**Review by YasielCabrera — APPROVED** [review]
lgtm

**Pull request merged**